debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1112,13 +1113,17 @@ public void readSchema(Tables tables, String databaseCatalog, String schemaNameP
 
             column.nativeType(resolveNativeType(column.typeName()));
             column.jdbcType(resolveJdbcType(columnMetadata.getInt(5), column.nativeType()));
-
+            setDefaultValue(column, columnMetadata.getString(13));
```

Comment:
Isn't there a risk for a data race here? How is it ensured that when reading a change event, we got the right default value, in case of changes?

Reply:
During snapshot the schema changes are either disallowed or the schema is locked. In case of `ALTER` it is necessary to follow the special SQL Server procedure that requires cooperation of admin for same schema evolution so this is ensured at orgnizational level.